beeai/openshift/deployment-mcp-gitlab.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mcp-gitlab
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: mcp-gitlab
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: mcp-gitlab
+        deployment: mcp-gitlab
+    spec:
+      containers:
+      - args:
+        - mcp_servers/gitlab_server.py
+        command:
+        - /usr/bin/python
+        env:
+        - name: SSE_PORT
+          valueFrom:
+            configMapKeyRef:
+              key: SSE_PORT
+              name: x-mcp-server
+        image: image-registry.openshift-image-registry.svc:5000/jotnar-prod/mcp-server@sha256:708bd0415631f3a552cb0171aa3f111f581b3d7ab7c19a505e38702845cb8219
```

Comment:
If the image gets deleted from registry, the SHA would become invalid. Is there a way to workaround it so that deployment may work from scratch?

Reply:
how is the gitlab server getting access to gitlab ?

Reply:
Yes, it is needed and is should be removed from the agents.